### PR TITLE
Modal View to EDIT an event that is not past

### LINKED ISSUE #134: Frontend: Modal View to EDIT an event that is not past
When I am logged in as an admin, I want to be able to EDIT the detail about a specific event so I can decide need to do something about it.

Create a Modal View Edit Event Detail for Admins like shown below

image

Note: Create the components with the appropriate style. Please check the wireframes https://www.figma.com/file/9gN79UmpDPrAZk5jY4V7Ww/Study-Group-Management-System?node-id=0%3A1

### PR DESCRIPTION
**Pull request by nuha90**
# Description

Modal View to EDIT an event that is not past

Fixes #134 

# Checklist

- [x] I have performed a self-review of my own code

### CODE REVIEW

File: src/client/components/EditEventsModal/EditEventsModal.stories.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    shape({
+      id: PropTypes.string,
+      value: PropTypes.string,
+    }),
+  ),
+};
+
+EditEvents.defaultProps = {
+  items: [],
+};
```
Comment (tpetrina) [inline src/client/components/EditEventsModal/EditEventsModal.stories.js:51]: Why create a new component inside a storybook file? This should not be here.

**Review by tpetrina — CHANGES_REQUESTED** [review]
Please remove extra code.

File: src/client/components/EditEventsModal/EditEventsModal.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    </div>
+  );
+}
+EditEventsModal.propTypes = {
+  heading: PropTypes.string,
+  path: PropTypes.string,
+  title: PropTypes.string,
+  datetime: PropTypes.string,
+  mentor: PropTypes.string,
+  link: PropTypes.string,
```
Comment (MaherHussain) [inline src/client/components/EditEventsModal/EditEventsModal.js:53]: i think you need also to define onSubmit prop . "onSubmit:PropTypes.func,"

File: src/client/components/EditEventsModal/EditEventsModal.stories.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      heading="Edit Event (Time Slot)"
+      path={ElasticLogo}
+      title="Elastic - Week 46"
+      datetime="Wednesday | 11 Nov 20|10:00 CEST"
+      mentor="name"
+      link="www.hyf.org"
+      disabled={emptyInput}
+    />
+  );
+};
```
Comment (tpetrina) [inline src/client/components/EditEventsModal/EditEventsModal.stories.js:37]: You still have a component in story that is not needed. Why don't you move all this code under the `EditNotPastEvent` below?

**Review by tpetrina — CHANGES_REQUESTED** [review]

**Review by tpetrina — APPROVED** [review]

**Review by yukafjaponesa — APPROVED** [review]

**Pull request merged**